testcases/MachineConfig.py

```diff
@@ -196,6 +202,17 @@ def runTest(self):
                 else:
                     self.fail("Failed to configure pmem")
 
+        if "nx_gzip=1" in self.machine_config:
```

Comment:
we should be checking  only nx_gzip and provided value is qos_credits just make sure we should doing round off